File: techila/examples/R/Tutorial/1_distribution/local_function.r
# Copyright 2010-2013 Techila Technologies Ltd.

# This function contains the locally executable function, which can be
# executed on the End-Users computer. This function does not
# communicate with the Techila environment.
#
# Usage:
# source("local_function.r")
# result <- local_function(x)
# x: the number of iterations in the for loop.
#
# Example:
# result <- local_function(5)

local_function <- function(x) {
 result <- array(0, dim = c(1, x))
 for (j in 1:x)
 result[1, j] <- 1 + 1
 result
}

File: techila/examples/python/Tutorial/1_distribution/local_function.py
# Copyright 2012-2013 Techila Technologies Ltd.

# This function contains the locally executable function, which can be
# executed on the End-Users computer. This function does not
# communicate with the Techila environment.
#
# Usage:
# execfile('local_function.py')
# result = local_function(x)
# x: the number of iterations in the for loop.
#
# Example:
# result = local_function(5)

def local_function(x):
    result = []
    for j in range(x):
        result.append(1 + 1)
    return result

